samples/cpp/calibration.cpp

```diff
@@ -339,7 +340,10 @@ int main( int argc, char** argv )
 
     cv::CommandLineParser parser(argc, argv,
         "{help ||}{w||}{h||}{pt|chessboard|}{n|10|}{d|1000|}{s|1|}{o|out_camera_data.yml|}"
-        "{op||}{oe||}{zt||}{a|1|}{p||}{v||}{V||}{su||}"
+        "{op||}{oe||}{zt||}{a||}{p||}{v||}{V||}{su||}"
+        "{ws|11|}"
+        "{fx||}{fy||}{cx||}{cy||}"
+        "{imshow-scale|1|}{enable-k3|1|}"
```

Comment:
In 3.4, by default `enable-k3=1` to keep the same behavior than before with: `flags|CALIB_FIX_K4|CALIB_FIX_K5`